### PR TITLE
[BUG] PMP-1739

### PR DESCRIPTION
**Pull request by dtiwarATS**
-- passed snapshot to pop-up component
-- contains fix for PMP-1651 as well

### CODE REVIEW

File: assets/src/components/parts/janus-popup/Popup.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ const Popup: React.FC<any> = (props) => {
   const [popupVisible, setPopupVisible] = useState(true);
   const [iconSrc, setIconSrc] = useState('');
 
+  const [initSnapshot, setInitSnapshot] = useState<any>();
```
Comment (ChristianMurphy) [inline assets/src/components/parts/janus-popup/Popup.tsx:25]: Is there a more specific interface than `any` that could be used here?

Comment (bsparks) [reply assets/src/components/parts/janus-popup/Popup.tsx:25]: maybe it could be `Record<string, any>`

Comment (ChristianMurphy) [reply assets/src/components/parts/janus-popup/Popup.tsx:25]: Or `Record<string, unknown>`

Comment (dtiwarATS) [reply assets/src/components/parts/janus-popup/Popup.tsx]: Done

**Review by ChristianMurphy — APPROVED** [review]

File: assets/src/components/parts/janus-popup/Popup.tsx (hunk 1 above)
Comment (dtiwarATS) [reply assets/src/components/parts/janus-popup/Popup.tsx]: I was planning using this

 snapshot: { [key: string]: string };

but something is happening with my Git. Hang On..

Comment (bsparks) [reply assets/src/components/parts/janus-popup/Popup.tsx]: `Record<string, string>` would be the same as that, but can't it have other values than string? `Record<string, unknown>` sounds good

**Review by bsparks — APPROVED** [review]

**Pull request merged**